moduleApplicationContext.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file is in the wrong place, what happened to your repo to have these files misplaced